buildscripts/CommonAssemblyInfo.cs

```diff
@@ -12,12 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
+using System.Runtime.InteropServices;
 using System.Reflection;
+using System.Security;
+
+[assembly: AssemblyProduct(
+	#if DOTNET45 // order is important
+	"Castle Core for .NET Framework 4.5"
+	#elif DOTNET40
+	"Castle Core for .NET Framework 4.0"
+	#elif DOTNET35
+	"Castle Core for .NET Framework 3.5"
+	#else
+	"Castle Core for .NET Core 1.0"
+	#endif
+)]

```

Comment:
Yep, one of the reasons I got rid of the dot between "Castle" and "Core", I'm setting `AssemblyProduct` and leaving `AssemblyTitle` empty. Being pragmatic the version resource is less important to me than the .NET module name in the assembly, and the file name.

Yep, don't really care that it is sort of meant to be for the client profile, client profile is so dead by now anyway.